File: src/main/java/com/likelionknu/applyserver/auth/data/dto/response/TokenResponseDto.java
package com.likelionknu.applyserver.auth.data.dto.response;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class TokenResponseDto {
    @JsonProperty("access_token")
    private String accessToken;

    @JsonProperty("refresh_token")
    private String refreshToken;

    private String name;
    private String role;
}

File: src/main/kotlin/com/likelionknu/applyserver/auth/data/dto/response/TokenResponseDto.kt
package com.likelionknu.applyserver.auth.data.dto.response

import com.fasterxml.jackson.annotation.JsonProperty

data class TokenResponseDto(
    @JsonProperty("access_token")
    val accessToken: String,

    @JsonProperty("refresh_token")
    val refreshToken: String,

    @JsonProperty("is_new_user")
    val isNewUser: Boolean = false,

    val name: String,
    val role: String
)